src/app/api/getDelegationPoints.ts

```diff
@@ -0,0 +1,49 @@
+import { encode } from "url-safe-base64";
+
+import { Pagination } from "../types/api";
+
+import { apiWrapper } from "./apiWrapper";
+
+export interface DelegationPoint {
+  staking_tx_hash_hex: string;
+  last_updated_btc_height: number;
+  staker: {
+    pk: string;
+    polkadot: number;
+  };
+  finality_provider: {
+    pk: string;
+    polkadot: number;
+  };
+}
+
+export interface PaginatedDelegationPoints {
+  data: DelegationPoint[];
+  pagination: Pagination;
+}
+
+export const getDelegationPoints = async (
+  key: string,
+  stakerBtcPk?: string,
+): Promise<PaginatedDelegationPoints> => {
+  if (!stakerBtcPk) {
+    throw new Error("Staker_btc_pk must be provided");
+  }
+
+  const params: Record<string, string> = {
+    pagination_key: encode(key),
+  };
+
+  if (stakerBtcPk) {
```

Comment:
We have already checked before that this is set so we don't need this `if` condition. We can directly set `staker_btc_pk` above.